var/spack/repos/builtin/packages/device-mapper/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class DeviceMapper(AutotoolsPackage):
+    """The device-mapper is a component of the linux kernel (since version
+    2.6) that supports logical volume management.  This package provides
+    the userspace configuration tools (dmsetup), library (libdevmapper) and
+    header files."""
+
+    homepage = "https://www.sourceware.org/dm"
+    url      = "https://sourceware.org/pub/lvm2/releases/LVM2.2.03.05.tgz"
+
+    version('2.2.03.05', sha256='ca52815c999b20c6d25e3192f142f081b93d01f07b9d787e99664b169dba2700')
```

Comment:
I think the package name is LVM2 and the version is 2.03.05. I would change the version numbers and add a `url_for_version`:

```python
def url_for_version(self, version):
    url = "https://sourceware.org/pub/lvm2/releases/LVM2.{0}.tgz"
    return url.format(version)
```

Reply:
agreed ...